govcd/nsxt_firewall_group_dynamic_security_group_test.go

```diff
@@ -0,0 +1,103 @@
+//go:build network || nsxt || functional || openapi || ALL
+// +build network nsxt functional openapi ALL
+
+package govcd
+
+import (
+	"github.com/vmware/go-vcloud-director/v2/types/v56"
+	. "gopkg.in/check.v1"
+)
+
+// Test_NsxtDynamicSecurityGroup tests out CRUD of Dynamic NSX-T Security Group
+//
+// Note. Dynamic Security Group is one type of Firewall Group. Other types are IP-Set and Static
+// Security Group.
+func (vcd *TestVCD) Test_NsxtDynamicSecurityGroup(check *C) {
+	skipNoNsxtConfiguration(vcd, check)
+	skipOpenApiEndpointTest(vcd, check, types.OpenApiPathVersion1_0_0+types.OpenApiEndpointFirewallGroups)
+
+	if vcd.client.Client.APIVCDMaxVersionIs("< 36.0") {
+		check.Skip("Dynamic security groups require VCD 10.3+")
+	}
+
+	adminOrg, err := vcd.client.GetAdminOrgByName(vcd.config.VCD.Org)
+	check.Assert(err, IsNil)
+
+	vdcGroup, err := adminOrg.GetVdcGroupByName(vcd.config.VCD.Nsxt.VdcGroup)
+	check.Assert(err, IsNil)
+
+	// vdcGroupEdgeGateway.
```

Comment:
Strange comment :)

Reply:
Yes. A leftover as I was planning on testing creation of Dynamic Security Groups using `vdcGroupEdgeGateway` entity, but in reality Dynamic Security Groups are bound to VDC Groups, not to Edge Gateways. Forgot to delete that and overlooked while self reviewing.